xysched/README.md

```diff
@@ -61,8 +72,9 @@ future.Finish(fmt.Printf, "the final value of x: %d\n", x)
 xysched.Now() <- future
 ```
 
-5. It is also possible to use `Then` and `Catch` methods to handle the returned
-value of `Future` or recover if it panicked.
+5.  It is also possible to use `Then` and `Catch` methods to handle the returned
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 2](https://app.codacy.com/gh/xybor/xyplatform/pullRequest?prid=10050498)
```suggestion
5. It is also possible to use `Then` and `Catch` methods to handle the returned
```